src/scss/components/_page-banner.scss

```diff
@@ -0,0 +1,63 @@
+ /**
+ * Page Banner
+ * =================================
+ * Example – Menu – Cuisine image at top of page.
+ *
+ * Used for content listing cards
+ *
+ * Documentation:
+ * https://fozzie.just-eat.com/styleguide/ui-components/page-banner
+ */
+
+@mixin pageBanner() {
+
+    .c-pageBanner {
+        background-color: $white;
+        margin-bottom: -15px;
+        max-height: 250px;
+        min-height: 100px;
+        overflow: hidden;
+        position: relative;
+
+        @include media('>=wide') {
+            max-height: 300px;
+        }
+
+        @include media('>=huge') {
+            max-height: 370px;
+        }
+
+        // Prevent IE from applying these styles as it can't correctly position the rays
+        @supports (position: absolute) {
+            &:after {
+                background-image: inline('#{$img-path}/decoration/rays--white--small.svg');
+                background-position: bottom left;
+                background-repeat: no-repeat;
+                bottom: -1px;
+                content: '';
+                height: 100%;
+                left: 0;
+                position: absolute;
+                width: 100%;
+
+                @include media('>mid') {
+                    background-image: inline('#{$img-path}/decoration/rays--white.svg');
+                }
+            }
+        }
+    }
+
+        .c-pageBanner--negativeBottom--aboveMid {
```

Comment:
Would be good to add a comment saying what this is for.  Seems like it's moving moving something below it up, but use-case would be good to include